src/types/onyx/Currency.ts

```diff
@@ -0,0 +1,12 @@
+type Currency = {
+    /** Symbol for the currency */
+    symbol?: string;
+
+    /** Name of the currency */
+    name?: string;
+
+    /** ISO4217 Code for the currency */
+    ISO4217?: string;
+};
```

Comment:
```suggestion
type Currency = {
    /** Symbol for the currency */
    symbol: string;

    /** Name of the currency */
    name: string;

    /** ISO4217 Code for the currency */
    ISO4217: string;

    /** Number of decimals the currency can have, if this is missing, we assume it has 2 decimals */
    decimals?: number;
};
```

Reviewing this as part of: https://github.com/Expensify/Expensify/issues/297946

Why are we specifying `symbol`, `name` and `ISO4217` as optional properties? I see that they are always there:

https://github.com/Expensify/App/blob/5f11116ab221b76d1bc43d1916c2630c00979470/tests/unit/currencyList.json

Also, the decimal property, which, as far as I know is being used, is missing,